src/audio/crossover/crossover.c

```diff
@@ -671,26 +671,26 @@ static int crossover_copy(struct comp_dev *dev)
 	else
 		num_sinks = num_assigned_sinks;
 
-	source = buffer_acquire_irq(source);
+	source = buffer_acquire(source);
 
 	/* Check if source is active */
 	if (source->source->state != dev->state) {
-		source = buffer_release_irq(source);
+		source = buffer_release(source);
```

Comment:
This  removal seems safer as the "_irq" has been done with a specific pattern and we can now review and conclude it is not needed and we can remove the variant.